ansible/playbooks/products/hand_e/desktop_icons/test_suite.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
this file should be in *tests* folder not in the root of the role

Reply:
Fixed in latest commit